package.json

```diff
@@ -13,7 +13,7 @@
     "lint:fix": "prettier --loglevel warn --write \"packages/**/*.{ts,tsx}\" && eslint \"packages/**/*.{ts,tsx}\" --quiet --fix",
     "release": "yarn workspace fastify-renderer publish",
     "prerelease": "yarn workspace fastify-renderer run gitpkg publish",
-    "test": "jest --forceExit",
+    "test": "jest --forceExit -w=1",
```

Comment:
parallel tests can cause issues locally

Reply:
@AyoubElk did you ever seen anything about this? Kristian and I were both seeing errors locally only sometimes when running with jest's parallel by default configuration.